tests/integration/UploadSlideBackTranslationIntegrationTest.php

```diff
@@ -0,0 +1,317 @@
+     *
+     * @return int
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     * @throws \Exception
+     */
+    public function testCreateNewStory(): void
+    {
+        // request payload
+        $payload = [
+            'Key' => '',
+            'PhoneId' => 'rstuvw',
+            'Data' => base64_encode('Test data for story file'),
+            'TemplateTitle' => $this->storyTemplate,
+            'IsWholeStory' => "true"
+        ];
+
+        $response = $this->httpClient->request("POST", self::$host . $this->uri, ['form_params' => $payload]);
+        $responseJson = $response->getBody()->getContents();
+
+        $this->assertEquals(200, $response->getStatusCode());
+        $responseArr = json_decode($responseJson, true);
+        $this->assertArrayHasKey('StoryId', $responseArr);
+        $storyId = $responseArr['StoryId'];
+
+
+        # verify story is created correctly in the database
+        $stories = $this->getStories($storyId);
+        $this->assertCount(1, $stories);
+
+        $this->assertEquals([
+            "id" => $storyId,
+            "title" => "test_story",
+            "language" => "",
+            "projectId" => $this->getProjectId($payload['PhoneId']),
+            "note" => "",
+            'FirstThreshold' => null,
+            'SecondThreshold' => null,
+        ], $stories[0]);
+
+
+        # verify slides are all created correctly in the database
+        $slides = $this->getSlides($storyId);
+        $this->assertCount(3, $slides);
+        $this->assertEquals(0, $slides[0]['slideNumber']);
+        $this->assertEquals(1, $slides[1]['slideNumber']);
+        # In test `story.json`, third slide type is `COPYRIGHT` which is skipped as per implementation
+        $this->assertEquals(3, $slides[2]['slideNumber']);
+
+        $storyFile = sprintf("%s/%s/%s/wholeStory.m4a", self::$uploadedProjectDir, $payload['PhoneId'], $storyId);
+        $this->assertFileExists($storyFile);
+        $this->assertEquals(file_get_contents($storyFile), base64_decode($payload['Data']));
+    }
+
+
+    /**
+     * @depends testCreateNewStory
+     *
+     * @return void
+     * @throws GuzzleException
+     * @throws \Exception
+     */
+    public function testUpdateExistingStory()
+    {
+        // payload for creating a new story
+        $payload = [
+            'Key' => '',
+            'PhoneId' => 'lmnopq',
+            'Data' => base64_encode('Test data for story file'),
+            'TemplateTitle' => $this->storyTemplate,
+            'IsWholeStory' => "true",
+            'Language' => 'es',
+        ];
+
+        $response = $this->httpClient->request("POST", self::$host . $this->uri, ['form_params' => $payload]);
+        $responseJson = $response->getBody()->getContents();
+        $this->assertEquals(200, $response->getStatusCode());
+
+        $responseArr = json_decode($responseJson, true);
+        $storyId = $responseArr['StoryId'];
+
+        # verify story is created correctly in the database
```

Comment:
In this test, we don't need to assert all these fields when creating a story because this is already tested by testCreateNewStory. Can you remove the assertions? Please keep the assertions after updating.